spec/test_collector/test_links/reporter_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'buildkite/test_collector/test_links_plugin/reporter'
+
+RSpec.describe Buildkite::TestCollector::TestLinksPlugin::Reporter do
+  let(:examples) { RSpec.world.filtered_examples }
```

Comment:
I've never seen this `RSpec.world` before... is it this thing? https://rspec.info/documentation/3.9/rspec-core/RSpec/Core/World.html If so, perhaps refactor to remove? 

I also don't know if you've seen this, but there are some helper methods to mock example objects in this file: https://github.com/buildkite/test-collector-ruby/blob/c57c586b03de84da7faf725e701ef8d55e2844db/spec/support/rspec_example_trace_helpers.rb

Reply:
Ah yup, I missed that warning! I can pretty much do the same thing using the helper method so have removed